docs/integrations/mirroring.md

```diff
@@ -112,6 +112,7 @@ def get_modified_remote_data_command(client, args):
 
     return GetModifiedRemoteDataResponse(modified_incident_ids)
 ```
+* **Last Mirror Run (Available from 6.6)** - get_last_mirror_run() is the function (in CommonServerPython) that retrieves the previous mirror run data that was set. Setting is done with the set_last_mirror_run() function (in CommonServerPython). Storing and getting this data (a dictionay) enables us controling the lastUpdate timestamp from the integration or any other data we want to save between the mirror runs.
```

Comment:
You can refer to this link https://xsoar.pan.dev/docs/reference/api/common-server-python#get_last_mirror_run
We generate docs for CommonServerPython

And just use the function in the code example.

Reply:
Done